documentation/src/main/asciidoc/user_guide/rolling_upgrades.adoc

```diff
@@ -1,130 +1,137 @@
 [[rolling_upgrades]]
-== Rolling upgrades
-Rolling upgrades is the process by which an {brandname} installation is
-upgraded without a service shutdown. In the case of {brandname} library/embedded mode, it refers to an
-installation to the nodes where {brandname} is running in library/embedded mode.
-For {brandname} servers, it refers to the server side components, not
-the client side. The upgrade could involve hardware change, or software
-change, such as upgrading the {brandname} version in use.
+== Performing Rolling Upgrades
 
-Rolling upgrades can be done in {brandname} installations using {brandname} in
-embedded or library mode, or in server mode. Here are the instructions for
-each use case:
+Upgrade {brandname} without downtime or data loss. You can perform rolling upgrades in Library Mode (embedded) or Remote Client/Server Mode to start using a more recent version of {brandname}.
```

Comment:
The CLI cache loader is gone. We do not support embedded rolling upgrades

Reply:
thanks @tristantarrant I've updated to remove RU procedures for embedded.